gnd/src/main/java/com/google/android/gnd/ui/map/MapAdapter.java

```diff
@@ -37,15 +37,21 @@
    */
   Observable<Point> getDragInteractions();
 
-  /** Returns a stream that emits the viewport center on each camera movement. */
-  Observable<Point> getCameraMoves();
+  /** Returns a stream that emits the viewport camera position on each camera movement. */
```

Comment:
Strike "viewport", since it implies the bounds are returned; better just "camera position".

Reply:
Thanks. Btw did you know that you can insert a suggestion like this using the +/- symbol above?